hypertrace-graphql-labels-schema/src/main/java/org/hypertrace/graphql/label/schema/mutation/LabelsMutationSchema.java

```diff
@@ -0,0 +1,37 @@
+package org.hypertrace.graphql.label.schema.mutation;
+
+import graphql.annotations.annotationTypes.GraphQLDataFetcher;
+import graphql.annotations.annotationTypes.GraphQLField;
+import graphql.annotations.annotationTypes.GraphQLName;
+import graphql.annotations.annotationTypes.GraphQLNonNull;
+import org.hypertrace.graphql.label.deserialization.LabelIdArgument;
+import org.hypertrace.graphql.label.deserialization.LabelKeyArgument;
+import org.hypertrace.graphql.label.mutator.LabelCreateMutator;
+import org.hypertrace.graphql.label.mutator.LabelDeleteMutator;
+import org.hypertrace.graphql.label.mutator.LabelUpdateMutator;
+import org.hypertrace.graphql.label.schema.Label;
+
+public interface LabelsMutationSchema {
+  String CREATE_LABEL = "createLabel";
+  String DELETE_LABEL = "deleteLabel";
+  String UPDATE_LABEL = "updateLabel";
+  String LABEL_ARGUMENT_NAME = "newLabel";
+
+  @GraphQLField
+  @GraphQLNonNull
+  @GraphQLName(CREATE_LABEL)
+  @GraphQLDataFetcher(LabelCreateMutator.class)
+  Label createLabel(@GraphQLNonNull @GraphQLName(LabelKeyArgument.ARGUMENT_NAME) String key);
```

Comment:
We should wrap the key argument into another graphql interface, since the create label currently only takes a key, but it could take other arguments in future

Reply:
Do I have to create a "Create label" type and make key as a member variable? Is this what you mean?